src/main/java/duke/Task.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+/**
+ * The <code>Task</code> class is an abstract class with a name and a boolean to represent whether
+ * it is completed.
+ *
+ * @author Zhu Yijie
+ */
+public abstract class Task {
+    protected String name;
+    protected boolean isCompleted;

```

Comment:
these could be set as private but okay

Reply:
Ah.... If I set as private, wouldn't Todo/Deadline/Event classes not be able to acccess them?